src/test/java/duke/parser/ParserTest.java

```diff
@@ -0,0 +1,230 @@
+package duke.parser;
```

Comment:
I like that you have a number of tests to ensure that your methods are working correctly!

Reply:
Thank you.